Modules/Tests/YosemiteTests/PointOfSale/PointOfSaleLocalSearchPurchasableItemFetchStrategyTests.swift

```diff
@@ -0,0 +1,530 @@
+import Foundation
+import Testing
+import Networking
+@testable import Storage
+@testable import Yosemite
+
+@Suite("PointOfSaleLocalSearchPurchasableItemFetchStrategy Tests")
+struct PointOfSaleLocalSearchPurchasableItemFetchStrategyTests {
+    private let siteID: Int64 = 123
+    private let searchTerm = "test"
+    private var grdbManager: GRDBManager!
+    private let variationsRemote = MockProductVariationsRemote()
+    private let mockAnalytics = MockPOSItemFetchAnalyticsTracking()
+
+    init() async throws {
+        grdbManager = try GRDBManager()
+
+        // Initialize site
+        try await grdbManager.databaseConnection.write { db in
+            try PersistedSite(id: siteID).insert(db)
+        }
```

Comment:
We get a Swift6 warning here due capturing siteID, which seems to be fixed by just making a local variable so captures the value instead of self:
```suggestion
        let siteIDLocalCopy = self.siteID
        try await grdbManager.databaseConnection.write { db in
            try PersistedSite(id: siteIDLocalCopy).insert(db)
        }
```